src/polyswarmd/bounties.py

```diff
@@ -52,37 +53,56 @@ def calculate_commitment(account, verdicts):
     return int_from_bytes(nonce), int_from_bytes(commitment)
 
 
+def get_assertion(guid, index, num_artifacts):
+    assertion = assertion_to_dict(
+        g.chain.bounty_registry.contract.functions.assertionsByGuid(guid.int, index).call(),
+        num_artifacts)
+
+    bid = [str(g.chain.bounty_registry.contract.functions.assertionBidByGuid(guid.int, index, j).call())
```

Comment:
Should we instead make a function that returns all the bids? This is going to hammer RPC

Reply:
Yea, I can look into that. 
Hopefully we still have space. 

I wish that I could just retrieve the whole array when something is mapped to it, but a function will have to do.